51b.md

```diff
@@ -0,0 +1,18 @@
+NIP-51b
+======
+
+Note Event List
+-------------------------
+
+`draft` `optional` `author:fiatjaf` `author:arcbtc` `author:monlovesmango` `author:eskema` 
+
+An event with kind `30001` is defined as a list event (see [NIP-51 - Lists](51.md)) for note events (or kind 1 events). The list of tags should follow the format for "e" tags as defined in [NIP-10 - Conventions for clients' use of e and p tags in text events](10.md).
+
+Commonly established list parameter names for this kind are:
+- `mute` - a list of tags that the user wants to mute
+- `follow` - a list of tags that the user wants to follow
+- `pin` - a list of tags that the user wants to pin (for display on profile)
+- `bookmark` - a list of tags that the user wants to bookmark
```

Comment:
```suggestion
- `mute` - a list of Note Events that the user wants to mute
- `follow` - a list of Note Events that the user wants to follow
- `pin` - a list of Note Events that the user wants to pin (for display on profile)
- `bookmark` - a list of Note Events that the user wants to bookmark
```

Reply:
again, it isn't a list of note events though. it a list of tags. the tag format is defined explicitly in the nip. 

"The list of tags should follow the format for "e" tags as defined in [NIP-10 - Conventions for clients' use of e and p tags in text events](https://github.com/monlovesmango/nips/blob/lists/10.md)."